FandomParser/InfoboxParser/InfoboxExtractor.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using FandomParser.Core;
+using InfoboxParser.Models;
+
+namespace InfoboxParser
+{
+    public class InfoboxExtractor : IInfoboxExtractor
+    {
+        private static readonly Regex regexReplaceEmpyLines = new Regex(@"^\s*$\n|\r", RegexOptions.Multiline | RegexOptions.Compiled);
```

Comment:
This regex matches empty lines, it does not replace them - could probably do with being renamed.